packages/kununu-footer/package.json

```diff
@@ -5,33 +5,25 @@
   "main": "dist",
   "author": "kununu",
   "license": "ISC",
-  "devDependencies": {
-    "@kununu/kununu-theme-v2": "2.0.2",
-    "babel-cli": "6.26.0",
-    "cross-env": "5.2.0",
-    "node-sass": "4.11.0",
-    "node-sass-module-importer": "0.1.0",
-    "nukleus": "14.1.2",
-    "react": "16.7.0",
-    "react-dom": "16.7.0",
-    "rimraf": "2.6.3"
```

Comment:
Don't we need this package for the `clean` script below?

Reply:
This was moved to the root package.json dependencies, lerna lets you install it there and then run commands that use it.